airflow/jobs/backfill_job.py

```diff
@@ -30,9 +30,7 @@
     AirflowException, DagConcurrencyLimitReached, NoAvailablePoolSlot, PoolNotFound,
     TaskConcurrencyLimitReached,
 )
-from airflow.executors.dask_executor import DaskExecutor
```

Comment:
This is the biggest problem. Not everyone uses Dask.

Reply:
Agree

Reply:
Yes agree, just had an issue istalling airflow from source:

`airflow db upgrade` gives
```
Traceback (most recent call last):
  File "/Users/feluelle/PycharmProjects/dmp/venv/bin/airflow", line 11, in <module>
    load_entry_point('apache-airflow', 'console_scripts', 'airflow')()
  File "/Users/feluelle/PycharmProjects/dmp/venv/src/airflow/airflow/__main__.py", line 40, in main
    args.func(args)
  File "/Users/feluelle/PycharmProjects/dmp/venv/src/airflow/airflow/cli/cli_parser.py", line 50, in command
    func = import_string(import_path)
  File "/Users/feluelle/PycharmProjects/dmp/venv/src/airflow/airflow/utils/module_loading.py", line 32, in import_string
    module = import_module(module_path)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/feluelle/PycharmProjects/dmp/venv/src/airflow/airflow/cli/commands/db_command.py", line 25, in <module>
    from airflow.utils import cli as cli_utils, db
  File "/Users/feluelle/PycharmProjects/dmp/venv/src/airflow/airflow/utils/db.py", line 26, in <module>
    from airflow.jobs.base_job import BaseJob  # noqa: F401 # pylint: disable=unused-import
  File "/Users/feluelle/PycharmProjects/dmp/venv/src/airflow/airflow/jobs/__init__.py", line 19, in <module>
    import airflow.jobs.backfill_job  # noqa
  File "/Users/feluelle/PycharmProjects/dmp/venv/src/airflow/airflow/jobs/backfill_job.py", line 33, in <module>
    from airflow.executors.dask_executor import DaskExecutor
  File "/Users/feluelle/PycharmProjects/dmp/venv/src/airflow/airflow/executors/dask_executor.py", line 22, in <module>
    from distributed import Client, Future, as_completed
ModuleNotFoundError: No module named 'distributed'

```